src/main/java/uk/gov/hmcts/reform/laubackend/idam/controllers/IdamLogonAuditController.java

```diff
@@ -62,29 +64,35 @@ public class IdamLogonAuditController {
     @Autowired
     private AppInsights appInsights;
 
-    @ApiOperation(tags = "POST end-point", value = "Save IdAM logon audits", notes = "This operation will "
+    @Operation(tags = "POST end-point", summary = "Save IdAM logon audits", description = "This operation will "
             + "persist IdAM logons entries which are posted in the request. Single IdAM LogonAudit per request will "
             + "be stored in the database.")
     @ApiResponses({
-            @ApiResponse(code = 201,
-                    message = "Created logonLog response - includes id from DB.",
-                    response = LogonLogPostResponse.class),
-            @ApiResponse(code = 400,
-                    message = "Invalid logon audit",
-                    response = LogonLogPostResponse.class),
-            @ApiResponse(code = 403, message = "Forbidden",
-                    response = LogonLogPostResponse.class),
-            @ApiResponse(code = 500, message = "Internal Server Error",
-                    response = LogonLogPostResponse.class)
+        @ApiResponse(
+            responseCode = "201",
+            description = "Created logonLog response - includes id from DB.",
+            content = { @Content(schema = @Schema(implementation = LogonLogPostResponse.class))}),
+        @ApiResponse(
+            responseCode = "400",
+            description = "Invalid logon audit",
+            content = { @Content(schema = @Schema(implementation = LogonLogPostResponse.class))}),
+        @ApiResponse(
+            responseCode = "403",
+            description = "Forbidden",
+            content = { @Content(schema = @Schema(implementation = LogonLogPostResponse.class))}),
+        @ApiResponse(
+            responseCode = "500",
+            description = "Internal Server Error",
+            content = { @Content(schema = @Schema(implementation = LogonLogPostResponse.class))})
     })
     @PostMapping(
-            path = "/audit/logon",
+            path = "/audit/logon/",
```

Comment:
The slash is unnecessary for both GET and POST.

Reply:
Please remove the slash because this change will most probably break the front-end.